Mathmagician.Tests/CommandTests.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using Mathmagician.Commands;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Mathmagician.Tests
+{
+    [TestClass]
+    public class CommandTests
+    {
+        [TestMethod]
+        public void AssertNumbersToStringReturnsString()
+        {
+            Command testCommand = new Command();
+            int[] testNumbers = { 1, 2, 3, 4, 5 };
+            testCommand.Numbers.AddRange(testNumbers);
+
+            Assert.IsTrue(testCommand.NumbersToString() is string);
+        }
+
```

Comment:
Actually, I was ultimately going to make command an abstract class that
couldn't be instantiated, wasn't sure how to go about testing that or if
you even would. As far as the string.format thing goes, I though string
builder was better when you were concatenating more than a couple times.
Can do though!

On Thu, Aug 4, 2016, 3:09 PM Kate <notifications@github.com> wrote:

> In Mathmagician.Tests/CommandTests.cs
> <https://github.com/nss-evening-cohort-3/mathmagician-group-project-dr-moneysack-and-friend-s/pull/7#discussion_r73593416>
> :
>
> > +
> > +namespace Mathmagician.Tests
> > +{
> > +    [TestClass]
> > +    public class CommandTests
> > +    {
> > +        [TestMethod]
> > +        public void AssertNumbersToStringReturnsString()
> > +        {
> > +            Command testCommand = new Command();
> > +            int[] testNumbers = { 1, 2, 3, 4, 5 };
> > +            testCommand.Numbers.AddRange(testNumbers);
> > +
> > +            Assert.IsTrue(testCommand.NumbersToString() is string);
> > +        }
> > +
>
> You'll need to test that you can create an instance of command, too.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/nss-evening-cohort-3/mathmagician-group-project-dr-moneysack-and-friend-s/pull/7/files/7e09d844874c8653a6ecb541525d0a6472e523dd#r73593416>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AOmzH7tDB8lUBSsTAklnibSVUYnEtH-Rks5qckbegaJpZM4JcRLz>
> .
>